src/main/java/com/google/sps/servlets/MentorEvidenceServlet.java

```diff
@@ -73,4 +84,37 @@ private void updateMentorEvidence(String paragraph) {
       logger.log(Level.SEVERE, exception.getMessage(), exception);
     }
   }
+
+  /**
+   * Adds a list of approvers (currently only the admins) to the mentor in the database.
+   */
+  private void addApprovers() {
+    int userId = Utility.getUserId();
+    // Create array to store IDs of approvers.
+    // TODO(oumontiel): Get IDs from all admins and remove hardcoded IDs.
+    int[] approvers = {1, 2, 3};
```

Comment:
Ideally these values would also be stored as constants in the SqlConstants file - They could be something like OMAR_USER_ID or something like that.

Reply:
Done!